src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -926,3 +928,198 @@ def acquireTiledArea(streams, stage, area, overlap=0.2, settings_obs=None, log_p
     executeAsyncTask(future, task.run)
 
     return future
+
+
+def acquireOverview(streams, stage, area, focus, ccd, overlap=0.2, settings_obs=None, log_path=None, zlevels=None,
+                    registrar=REGISTER_GLOBAL_SHIFT, weaver=WEAVER_MEAN, focusing_method=FocusingMethod.NONE):
+    """
+    Start autofocus and tiled acquisition tasks for the given list of areas.
+
+    :param streams: (list of Stream) the streams to acquire
+    :param stage: (Actuator) the stage to move
+    :param area: (list of area) area is defined by 4 floats :left, top, right, bottom positions
+     of the acquisition area (in m)
```

Comment:
this explanation is not very clear, is this the list of areas you want to acquire? If yes, can you update the description to explain that? And does it then make sense to rename the argument from `area` to `areas`?